types/ramda/index.d.ts

```diff
@@ -1402,7 +1402,7 @@ export function pickAll(names: readonly string[]): <T, U>(obj: T) => U;
 /**
  * Returns a partial copy of an object containing only the keys that satisfy the supplied predicate.
  */
-export function pickBy<T, U>(pred: ObjPred, obj: T): U;
+export function pickBy<T, U>(pred: ObjPred<T>, obj: T): U;
 export function pickBy(pred: ObjPred): <T, U>(obj: T) => U;
```

Comment:
Should you not also add the generic to `ObjPred` in the curried form too?

Reply:
👌 I have fixed it.